test/components/VariantSelector/VariantSelector.spec.tsx

```diff
@@ -19,19 +19,46 @@ describe("VariantSelector", () => {
 
     addHandlers(
```

Comment:
extract this logic into it's own module and use it in VariantSelector.spec.tsx and SimpleCard.spec.tsx